observation_portal/requestgroups/test/test_api.py

```diff
@@ -1196,7 +1196,7 @@ def test_cadence_invalid_period(self):
         bad_data['requests'][0]['cadence']['period'] = -666
         response = self.client.post(reverse('api:request_groups-cadence'), data=bad_data)
         self.assertEqual(response.status_code, 400)
-        self.assertEqual(response.json()['cadence']['period'], ['Ensure this value is greater than or equal to 0.02.'])
+        self.assertEqual(response.json()['requests'][0]['cadence']['period'], ['Ensure this value is greater than or equal to 0.02.'])
```

Comment:
I updated this test because I have updated the cadence endpoint to have an upper level serializer with a `requests` field:

```
class CadenceRequestGroupSerializer(serializers.Serializer):
    requests = import_string(settings.SERIALIZERS['requestgroups']['CadenceRequest'])(many=True)
```

This gives us accurate auto-documentation without changing our core API.

When this endpoint returns an error now, it has one extra level in the return dict. I am hoping this doesn't interfere with anything in the frontend?

Reply:
It shouldn't. Looks like the frontend doesn't actually display the specific error, just that something when wrong